pkgs/applications/finance/denaro/default.nix

```diff
@@ -1,6 +1,7 @@
 { lib
 , buildDotnetModule
 , fetchFromGitHub
+, fetchpatch
```

Comment:
```suggestion
```

Reply:
fixed in https://github.com/NixOS/nixpkgs/pull/254157/commits/331df1911c45ed8ce698a6736ae08d00aae7dab0